.flake8

```diff
@@ -0,0 +1,8 @@
+[flake8]
+ignore = F403
```

Comment:
Why is this needed? We don't have any usage of `from module import *` afaict

Reply:
Oops will remove it